README.md

```diff
@@ -1 +1,5 @@
-# Review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Update your readme to contain the information about the project!

Reply:
- [ ] Kindly phrase your comments in a polite and considerate manner, like a suggestion, instead of a command. This will make it easy for the student to accept and implement them. :wink: